pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -404,7 +405,7 @@ class HardwarePulse:
     internally in PL to group into a single object all the data related to a single EM field.
     Args:
         amplitude (Union[float, Callable]): float or callable returning the amplitude of an EM
-            field
+            field.
```

Comment:
Do we need?

Reply:
Nope, don't remember when that happened or why. Will remove.